tests/request/test_request.py

```diff
@@ -72,7 +73,7 @@ async def make_assertion(*args, **kwargs):
 
 @pytest.fixture()
 async def httpx_request():
-    async with HTTPXRequest() as rq:
+    async with NonchalantHttpxRequest() as rq:
```

Comment:
is this a change due to py3.13 or is this rather a stability improvement? If so, I would like to keept it for #4271, tbh …

Reply:
I see. NonchalantHR xfails on timeout and retry-after, though … I guess for the cases were we monkeypatch the request anyways, it doesn't matter, but please double check if there any new usages of `NHR` where we should actually better care to always make a real request

Reply:
just checked, none of them make an actual request.